example/src/main/java/io/netty/example/mqtt/heartBeat/MqttHeartBeatBrokerHandler.java

```diff
@@ -27,6 +28,7 @@
 import io.netty.handler.timeout.IdleState;
 import io.netty.handler.timeout.IdleStateEvent;
 
+@Sharable
 public class MqttHeartBeatBrokerHandler extends ChannelInboundHandlerAdapter {
```

Comment:
as this one s now sharable you may want to consider to add a private contructor, mark it final and use an static instance.